runsc/boot/config.go

```diff
@@ -201,6 +201,12 @@ type Config struct {
 
 	// AlsoLogToStderr allows to send log messages to stderr.
 	AlsoLogToStderr bool
+
+	// AggressiveCache enables page cache filling during read/write
```

Comment:
Can you elaborate on the behavior of `AggressiveCache`? In particular, how does it differ from `inode_cached.go`'s fillCache selection? It seems like it's a combination of:

- Force use of the sentry page cache instead of the host page cache [1], which is already achievable via `fs.MountSourceFlags.ForcePageCache` (although this isn't plumbed out to runsc yet, doing so would be preferable to this new option)

- Do not make the inserted segment visible to `pgalloc.MemoryFile`, which is not acceptable (any reclaim mechanism you add must interoperate cleanly with `pgalloc`-driven eviction, not be independent of it)